src/main/java/adris/altoclef/util/baritone/GoalDirectionXZ.java

```diff
@@ -1,46 +1,52 @@
 package adris.altoclef.util.baritone;
 
+
 import baritone.Baritone;
 import baritone.api.BaritoneAPI;
 import baritone.api.pathing.goals.Goal;
 import net.minecraft.util.math.Vec3d;
 
+
 public class GoalDirectionXZ implements Goal {
-    public final double originx;
+    // TODO: 2021-05-22 why public...
```

Comment:
You're right, should be private. (I copied from Baritone's "GoalStrictDirection" which has public variables, accidentally left that in)